updater/reports/ReportRepoUsage.py

```diff
@@ -13,6 +13,51 @@ def updateDailyData(self):
 		self.data.extend(newData)
 		self.truncateData(self.timeRangeTotal())
 		self.sortDataByDate()
+		self.detailedHeader, self.detailedData = self.parseData(self.executeQuery(self.detailedQuery()))
+
+
+	def detailedQuery(self):
+		query = '''
+			SELECT
+				org, repo
+			FROM (
+		'''
+		query += self.activeReposQuery()
+		query += '''
+			) AS active
+			INNER JOIN
+			(
+				SELECT repository_id, COUNT(*) AS count
+				FROM pushes
+				GROUP BY repository_id
+			) AS push_tally
+			ON active.repository_id = push_tally.repository_id
+			ORDER BY count DESC
+		'''
+		return query
```

Comment:
Nit: In `activeReposQuery` below you assign `query` once. Here you append to the `query` variable. I would only use one stile. I think I prefer "assign once".

Reply:
@larsxschneider: I find both styles okay as they are used here. Simple concatenations with `+` are fine for single variables etc., while entire lines (such as `self.activeReposQuery()` are broken down with `query +=`. I think it’s fine as it is 😄.